### ISSUE TITLE
[ldd-rings] Deprecate copied classes from Geometry and Display dictionaries

### ISSUE THREAD

**Issue by mace-space** [pds-data-dictionaries/PDS4-LDD-Issue-Repo#242]
**Describe the issue identified (if applicable)**

The following classes exist in the Geometry and Display dictionaries and should not be copied in the Rings dictionary:

  - Body_Ident_Base
  - Reference_Frame_Ident
  - Central_Body_Ident
  - Display_Direction

**Describe the solution you'd like**

Remove these modified classes from the Rings dictionary (after ensuring that any useful modifications after they were copied into the Rings dictionary will not be lost). PDS4 labels can simply reference the classes from the geom and disp namespaces.

**Comment by matthewtiscareno** [comment]
Before we can deprecate these, we must be able to access them in their parent dictionaries.  Some of them are not yet exposed, and that must be fixed.

In fact, we don't have a good way to fix it right now, because the only way to expose an element is to expose it to all data providers, which allows it to be used without any context.  We need a way for elements to be exposed for use in other dictionaries but not directly to data providers.

**Comment by matthewtiscareno** [comment]
Update: There is now a SCR before the DDWG that would allow elements to be exposed for other LDDs but not for primary use in labels.  See [CCB#41](https://github.com/NASA-PDS/PDS4-CCB/issues/41). 

Also, it seems that not all of the base classes that we copied from Geometry LDD are still in the state that they were when we copied them, which means that some work is required before we can turn those duplications into references (e.g., `Body_Ident_Base` ==> `geom.Body_Ident_Base`).

**Comment by matthewtiscareno** [comment]
We want to move forward with exposing `Body_Ident_Base`, `Reference_Frame_Ident`, and `Central_Body_Ident` within the Geometry LDD, which is to say setting `element_flag = true`.  All three of these should also have `abstract_flag = true`, which is not currently the case for all of them.  Finally, our practice within the Rings LDD has been to allow an optional `local_identifier` for elements such as this, so we suggest adding that to these three. 

Once this has been done, we will proceed to build our own classes within the Rings LDD using these parents.

**Comment by matthewtiscareno** [comment]
This fix is on hold and will not be included in [v1.13](https://github.com/pds-data-dictionaries/ldd-rings/pull/28), because of an LDDTool bug that prevents Rings LDD from referencing Geometry LDD in the way we want to do it.  @jordanpadams says this will be fixed.
